### ISSUE TITLE
Automatic iframe height

### ISSUE THREAD

**Issue by Langmans** [Zequez/FactorioMods#6]
This depends on factorioforums.com offcourse, but you might like to use https://github.com/davidjbradshaw/iframe-resizer to keep the forum topic the correct height :)

Maybe it's possible with a simple setInterval however.

**Comment by Zequez** [comment]
Yeah, I found that library while trying this same thing, but as you say, they would also need to run a script on the forum for it to work. But it's not completely out of the question, they did like the project, so it's not too far of a stretch.
